Complex.mo

```diff
@@ -94,24 +94,23 @@ operator record Complex "Complex number with overloaded operators"
 </html>"));
     end multiply;
 
-    function scalarProduct "Scalar product c1*c2 of two complex vectors"
+    function scalarProduct "Scalar product of two complex vectors c1 and c2"
       import Complex;
       input Complex c1[:] "Vector of Complex numbers 1";
       input Complex c2[size(c1,1)] "Vector of Complex numbers 2";
-      output Complex c3 "= c1*c2";
+      output Complex c3 "= Scalar product of c1 and c2";
```

Comment:
IMO the equal sign "=" can be deleted here (and so the space right behind it).

Reply:
agreed

Reply:
That's really a minor problem ...